app/containers/Settings/Settings.jsx

```diff
@@ -32,6 +32,7 @@ import CogIcon from '../../assets/icons/cog-icon.svg'
 import NodeSelectIcon from '../../assets/icons/node-select.svg'
 import TimeIcon from '../../assets/icons/time-icon.svg'
 import SaveIcon from '../../assets/icons/save-icon.svg'
+import pack from '../../../package.json'
```

Comment:
very minor, but it would be better to call this import `packageJSON` or `packageJson`.

Reply:
👍 updated to just use a destructured import thanks for calling this out